frontend/packages/local-storage-operator-plugin/src/components/disks-list/disks-list-page.tsx

```diff
@@ -11,13 +11,14 @@ import {
 } from '@console/internal/components/factory';
 import { sortable } from '@patternfly/react-table';
 import { FirehoseResult, humanizeBinaryBytes, Kebab } from '@console/internal/components/utils';
-import { referenceForModel, NodeKind, K8sResourceCommon } from '@console/internal/module/k8s';
+import { referenceForModel, NodeKind } from '@console/internal/module/k8s';
 import { RowFilter } from '@console/internal/components/filter-toolbar';
 import { useFlag } from '@console/shared/src/hooks/flag';
 import { OCS_ATTACHED_DEVICES_FLAG } from '@console/ceph-storage-plugin/src/features';
```

Comment:
This should not be imported from ceph-storage-plugin as this will create circular dependency. Please create a constant in LSO and use it. You can import the constant from LSO to Ceph plugin  too if you want avoid redundancy.

Reply:
Thanks for specifying. I was thinking it another way .(like noobaa, where ceph is parent).
Its the only reason I did not move this file to ceph. Now I have one.
Thanks @gnehapk for pointing

Reply:
@cloudbehl  this ^^ applies to your concern as well  https://github.com/openshift/console/pull/6168#issuecomment-666913659